playbooks/roles/ssh-forward/templates/instructions.md.j2

```diff
@@ -19,6 +19,8 @@ SSH Tunnel
    * Port `443` is available as a fallback option if you are on a network that restricts access to the default SSH port.
 1. Go to Connection --> Data.
 1. Enter `forward` in the *Auto-login username* field.
+1. Go to Connection --> SSH.
+1. Check `Don't start a shell or command at all`.
```

Comment:
Could you also add a small section on what the error looks like if you forget this step? Something like "If you see the message "this account is for port forwarding only" ensure you have selected "Don't start a shell or command at all" in your Putty configuration". ?

Reply:
Maybe this should be at the very bottom of the instructions so that it can reference the Linux solution as well. "If you see xxx then make sure you are providing -N in your ssh command line`.

Reply:
@cpu I have added some clarification at the end of the documentation. The French part is machine translated.